src/document.h

```diff
@@ -70,6 +70,9 @@ typedef struct GeanyFilePrefs
 GeanyFilePrefs;
 
 
+#define GEANY_TYPE_DOCUMENT (document_get_type())
+GType document_get_type (void);
```

Comment:
Not much, but if they aren't needed by GIR and are only used internally, they probably be better off in the `GEANY_PRIVATE` section.